### DISCUSSION TITLE
Display images in Readme and in Github pages

### DISCUSSION THREAD

**Discussion by mtouzot** [git-lfs/git-lfs#5256] [Q&A]
Hi !

I am facing a problem by using git lfs. I recently track image files in my [GameBEye](https://github.com/mtouzot/GameBEye) repo. All tracked images are displayed in the readme. And it's not the same in the Github page (nothing but the path is displayed). I suppose it is due to the way git lfs stores the files but is there a way to avoid this ?

Thanks

**Comment by mtouzot** [comment]
It seems working updating the GitHub action linked to GitHub Pages creation by having :

- name: "Checkout"
  uses: actions/checkout@v3
  with:
    lfs: true

**Reply by bk2204** [reply]
I'm glad you got this figured out.  I don't think it was possible to use LFS for the pre-Actions Pages, but I'm glad to know that this is now possible if you're using the Actions-based version.
